### ISSUE TITLE
Resume failed download instead of requiring uninstallation first

### ISSUE THREAD

**Issue by mlt** [hakuna-m/wubiuefi#23]
I was trying to give it a try (for the first time). However I got the message box saying
```
---------------------------
Xubuntu Installer
---------------------------
An error occurred:

Socket Error: [Errno 10054] An existing connection was forcibly closed by the remote host

For more information, please see the log file: c:\users\mlt~1.des\appdata\local\temp\wubi-16.10-rev314.log
---------------------------
OK   
---------------------------
```
The content of the log is [available on hastebin](http://hastebin.com/mujukayime.tex). So perhaps the internet connection got dropped (though why log says download finished ❓😕).
When I restart the installer, it asks me to uninstall whatever is there and start all over (I tried twice).

It would be nice if download was resumed where it left off.... if it was a download issue.

P.S. I was using the latest available binary from *Releases*.

**Comment by hakuna-m** [comment]
wubi1610r314.exe is a pre-release with download links for 16.10 Beta 2. Additionally, the download links has been changed in the last hours for final release of 16.10.

For 16.10 we try to use only Beta 2 and final release instead of daily builds to get more stable download links. But this is not always possible.

wubi1610r315.exe is coming soon, source is already available. At the moment we test if the new download links are working.

**Comment by mlt** [comment]
Thank you for getting back so promptly :raised_hands: Indeed it went better with the latest release. Although it was strange that step for installing grub2 withing xubuntu seemed to take forever followed by removal of some packages and rolling back to something as far as I could read :confused: ... Nevertheless, upon few reboots into Windows, and realizing :bulb: I needed to enter UEFI boot menu, I saw Xubuntu and booted fine into it.

So I guess this issue is more about providing sane error messages so newcomers aren't that confused :sunglasses: Something like "File not found on remote host perhaps revision was changed" would be a far better message to a user than generic one. And a dumb reminder to check UEFI boot order would be also helpful:-)

**Comment by hakuna-m** [comment]
> Although it was strange that step for installing grub2 within xubuntu seemed to take forever followed by removal of some packages and rolling back to something as far as I could read :confused:

You are right. It is confusing. But it is sometimes a fast method to install pre-defined groups of packages followed by removal of unneeded packages. e.g. You don't need all language packages  which are on downloaded iso.

> Nevertheless, upon few reboots into Windows, and realizing :bulb: I needed to enter UEFI boot menu, I saw Xubuntu and booted fine into it. 

It would be easy to change this behaviour. But the behaviour has a reason. You should be always able to boot into Windows, because an unbootable Windows is the worst case for a lot of users. But of course, it would be a good idea to improve our Wiki.

> Something like "File not found on remote host perhaps revision was changed" would be a far better message to a user than generic one.

You are right. But there should be no changes of download links after final release and before final release it is not really recommanded for newcomers to use it.
